substrate/bin/node-template/runtime/src/lib.rs

```diff
@@ -153,7 +153,41 @@ parameter_types! {
 
 // Configure FRAME pallets to include in runtime.
 
+#[derive_impl(frame_system::config_preludes::SolochainDefaultConfig as frame_system::DefaultConfig)]
 impl frame_system::Config for Runtime {
+	/// The following are being injected by [`derive_impl`](`frame_support::derive_impl`) from
+	/// [`SoloChainDefaultConfig`](`struct@frame_system::config_preludes::SolochainDefaultConfig`),
+	/// but can be overridden if needed.
+	///
+	/// The ubiquitous event type.
+	/// type RuntimeEvent = RuntimeEvent;
+	///
+	/// The aggregated dispatch type that is available for extrinsics.
+	/// type RuntimeCall = RuntimeCall;
+	///
+	/// The ubiquitous origin type.
+	/// type RuntimeOrigin = RuntimeOrigin;
+	///
+	/// Converts a module to the index of the module in `construct_runtime!`.
+	///
+	/// This type is being generated by `construct_runtime!`.
+	/// type PalletInfo = PalletInfo;
+	///
+	/// What to do if a new account is created.
+	/// type OnNewAccount = ();
+	///
+	/// What to do if an account is fully reaped from the system.
+	/// type OnKilledAccount = ();
+	///
+	/// The set code logic, just the default since we're not a parachain.
+	/// type OnSetCode = ();
+	///
+	/// The hashing algorithm used.
+	/// type Hashing = BlakeTwo256;
+	///
+	/// Weight information for the extrinsics of this pallet.
+	/// type SystemWeightInfo = ();
+
```

Comment:
IMO this should be documented on top of the `SoloChainDefaultConfig`

Reply:
@bkchr Updated it. Thanks!